keras/src/trainers/epoch_iterator.py

```diff
@@ -75,55 +75,86 @@ def __init__(
     def _get_iterator(self):
         return self.data_adapter.get_numpy_iterator()
 
-    def enumerate_epoch(self):
```

Comment:
This method must be kept for backwards compatibility (for users who have a custom fit())

Reply:
It's back.